src/Eccube/Resource/template/install/migration_frame.twig

```diff
@@ -32,10 +32,6 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
     <link href="{{ publicPath }}template/install/dist/css/bootstrap.min.css?v={{ constant('Eccube\\Common\\Constant::VERSION') }}" rel="stylesheet">
     <link href="{{ publicPath }}template/install/assets/css/dashboard.css?v={{ constant('Eccube\\Common\\Constant::VERSION') }}" rel="stylesheet">
     <script src="{{ publicPath }}template/install/assets/js/ie10-viewport-bug-workaround.js?v={{ constant('Eccube\\Common\\Constant::VERSION') }}"></script>
-    <!--[if lt IE 9]>
```

Comment:
when  keep( support ) it, We have to download to local 3 these files.
checking EC-CUBE only support IE 10 above, so delete these.